### PR TITLE
K.krupa/rerender on resize and bugfix

### PR DESCRIPTION
**Pull request by karolk98**

### CODE REVIEW

File: RayTracer/Source/Materials/Texture.cs [hunk 1]
```diff
@@ -116,32 +116,25 @@ public void Write(string path)
...
                 raw);
 
-            GL.TexParameter(TextureTarget.Texture2D, TextureParameterName.TextureMinFilter,
-                (int) TextureMinFilter.Linear);
-            GL.TexParameter(TextureTarget.Texture2D, TextureParameterName.TextureMagFilter,
-                (int) TextureMagFilter.Linear);
-
-            GL.TexParameter(TextureTarget.Texture2D, TextureParameterName.TextureWrapS, (int) TextureWrapMode.Repeat);
-            GL.TexParameter(TextureTarget.Texture2D, TextureParameterName.TextureWrapT, (int) TextureWrapMode.Repeat);
-            GL.TexParameter(TextureTarget.Texture2D, TextureParameterName.TextureWrapR, (int) TextureWrapMode.Repeat);
```
Comment (tomasz-herman) [inline RayTracer/Source/Materials/Texture.cs:141]: Why are texture params removed?

File: RayTracer/Source/Materials/Texture.cs [hunk 2]
```diff
@@ -116,32 +116,25 @@ public void Write(string path)
...
+                PixelInternalFormat.Rgb,
                 Width,
                 Height,
                 0,
                 PixelFormat.Rgb,
                 PixelType.UnsignedByte,
                 raw);
 
-            GL.TexParameter(TextureTarget.Texture2D, TextureParameterName.TextureMinFilter,
-                (int) TextureMinFilter.Linear);
```
Comment (tomasz-herman) [inline RayTracer/Source/Materials/Texture.cs:135]: set min filter to GL_LINEAR_MIPMAP_LINEAR

File: RayTracer/Source/Materials/Texture.cs (hunk 1 above)
Comment (tomasz-herman) [inline RayTracer/Source/Materials/Texture.cs:141]: WrapR is only useful for 3d textures

Comment (karolk98) [reply RayTracer/Source/Materials/Texture.cs:141]: Removing it didnt cause any errors so i assumed they are not needed

**Review by tomasz-herman — APPROVED** [review]

**Pull request merged**